package.json

```diff
@@ -11,45 +11,45 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "~16.0.1",
-    "@angular/common": "~16.0.1",
-    "@angular/compiler": "~16.0.1",
-    "@angular/core": "~16.0.1",
-    "@angular/forms": "~16.0.1",
-    "@angular/platform-browser": "~16.0.1",
-    "@angular/platform-browser-dynamic": "~16.0.1",
-    "@angular/router": "~16.0.1",
+    "@angular/animations": "~17.3.1",
+    "@angular/common": "~17.3.1",
+    "@angular/compiler": "~17.3.1",
+    "@angular/core": "~17.3.1",
+    "@angular/forms": "~17.3.1",
+    "@angular/platform-browser": "~17.3.1",
+    "@angular/platform-browser-dynamic": "~17.3.1",
+    "@angular/router": "~17.3.1",
     "rxjs": "~7.8.1",
     "tslib": "^2.5.0",
-    "zone.js": "~0.13.0"
+    "zone.js": "~0.14.4"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "~16.0.1",
-    "@angular-eslint/builder": "16.0.1",
-    "@angular-eslint/eslint-plugin": "16.0.1",
-    "@angular-eslint/eslint-plugin-template": "16.0.1",
-    "@angular-eslint/schematics": "16.0.1",
-    "@angular-eslint/template-parser": "16.0.1",
-    "@angular/cli": "~16.0.1",
-    "@angular/compiler-cli": "~16.0.1",
-    "@angular/language-service": "~16.0.1",
+    "@angular-devkit/build-angular": "~17.3.2",
```

Comment:
Could you change to ~17.0.0 on all Angular packages?

Reply:
Will do.